web/pdf_rendering_queue.js

```diff
@@ -93,7 +93,10 @@ class PDFRenderingQueue {
     }
 
     if (this.onIdle) {
-      this.idleTimeout = setTimeout(this.onIdle.bind(this), CLEANUP_TIMEOUT);
+      this.idleTimeout = window.setTimeout(
+        this.onIdle.bind(this),
+        CLEANUP_TIMEOUT
+      );
```

Comment:
Please revert this change, since it'd be a case of the TypeScript definitions dictating how we can/can't write JavaScript code. Adding a one-liner comment with correct `type`-information actually seem preferable here.

Note also that we've got `setTimeout`-calls in various parts of the code-base and having one special case like this doesn't feel right (and I *really* don't want to convert all of them).